packages/tre/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://laurikari.net/tre/
+TERMUX_PKG_DESCRIPTION="Approximate regex matching library and agrep CLI tool"
+TERMUX_PKG_LICENSE="BSD-2-CLAUSE"
+TERMUX_PKG_VERSION=2021.03.21
+TERMUX_PKG_SRCURL=https://github.com/laurikari/tre/archive/6092368aabdd0dbb0fbceb2766a37b98e0ff6911.tar.gz
```

Comment:
Any particular reason for choosing this date and commit?

Reply:
Nothing other than its the SRCURL of the gentoo package version I use. And the [cross arch stability is good](https://packages.gentoo.org/packages/dev-libs/tre) . I can't be as sure with the current git version.

Reply:
Packages built against a specific commit usually split out the commit into a separate variable from the SRCURL.
```suggestion
_COMMIT=6092368aabdd0dbb0fbceb2766a37b98e0ff6911
TERMUX_PKG_SRCURL=https://github.com/laurikari/tre/archive/${_COMMIT}.tar.gz
```
For example `cairo-dock-core`:
https://github.com/termux/termux-packages/blob/e7c982d40e86a28fc10967b4af9a9782b8bb7faa/x11-packages/cairo-dock-core/build.sh#L5-L8

Reply:
I have force pushed some fixes to this PR.